utils/createOrders.ts

```diff
@@ -98,6 +86,8 @@ export class IexecOrders {
             datasetOrder: this.dataset,
             workerpoolOrder: this.workerpool,
             requesterOrder: this.requester,
+            // An alias for convenience
+            requestOrder: this.requester,
```

Comment:
```suggestion
            requestOrder: this.requester,
```
Can you replace the `requesterOrder` form (which is not quite appropriate) to the correct `requestOrder` form? (here or a todo note)

Reply:
Added a note here [Add todo note](https://github.com/iExecBlockchainComputing/PoCo/pull/123/commits/ca4a062f842c1628138e32cfe068a90e248c2a68)